regolith/helpers/a_presentationhelper.py

```diff
@@ -29,36 +30,36 @@ def subparser(subpi):
     subpi.add_argument("name", help="name of the presentation, meeting name if meeting,"
                                     "department if seminar",
                        )
-    subpi.add_argument("-t","--type", help=f"types, from {ALLOWED_TYPES}. Default is other",
-                       default="invited"
-                       )
-    subpi.add_argument("-d", "--begin_date",
-                       help="Input begin date for this expense. "
-                            "In YYYY-MM-DD format. Defaults to today's date",
+    subpi.add_argument("begin_date",
+                       help="Input begin date for this expense "
+                            "in YYYY-MM-DD format",
                        )
-    subpi.add_argument("-e,", "--end_date",
-                       help="Input end date for this expense. "
-                            "In YYYY-MM-DD format. Defaults to today's date",
+    subpi.add_argument("end_date",
+                       help="Input end date for this expense"
+                            "in YYYY-MM-DD format" ,
                        )
     subpi.add_argument("-p", "--person",
-                       help="the person submitting the presentation, used for presentation name",
-                       default="sbillinge"
+                       help="the person submitting the presentation, used for presentation name,"
```

Comment:
delete "used for presentation name``.  Change "person" to "person or people giving the presentation" (I think it can accept a list, right?).  Change ``--person`` to ``--people``, allow ``nargs = '+'``

Reply:
wait, I see the problem here.   We have an ``--authors`` below.  So we should delete ``--person`` altogether and everything I wrote above should apply to ``--authors`` and not to ``--person`` (i.e., so set the default author to ``default_user_id``.

Reply:
I believe person arg is used to to build the name of the presentation, eg if the person is sbillinge then the name will be 2007sb_something. However, if we use a list what should determine the name of the expense?